inc/namespace.php

```diff
@@ -598,3 +611,34 @@ function on_aws_guzzle_request_stats( TransferStats $stats ) {
 	}
 	send_trace_to_daemon( $trace );
 }
+
+function redact_metadata( $metadata ) {
+
+	$redact_keys_default = [];
+	if ( function_exists( 'apply_filters' ) ) {
+		$redact_keys_default = apply_filters( 'aws_xray.redact_metadata_keys', $redact_keys_default );
+	}
+
+	$redact_keys_required = [
+		'$_POST' => [
+			'pwd',
+		],
+	];
+
+	$redact_keys = array_merge( $redact_keys_default, $redact_keys_required );
```

Comment:
I don't think this is going to recursively merge keys, so not sure this will work when other things try add to $_POST

Reply:
Ah, quite correct, thank you! I'll fix that.